### PR TITLE
Validate $CORN

### PR DESCRIPTION
**Pull request by ErikThiart**
Added $corn

# Validate [$CORN](https://solscan.io/token/6DSqVXg9WLTWgz6LACqxN757QdHe1sCqkUfojWmxWtok)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: [https://twitter.com/CornOnSolana/status/1728499118530510992](https://twitter.com/CornOnSolana/status/1728499118530510992)
- Coingecko/ CMC URL (If available): [https://www.coingecko.com/en/coins/solanacorn](https://www.coingecko.com/en/coins/solanacorn)

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [x] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

**Comment by ErikThiart** [comment]
@Timoon21 do let me know if there is amything else you need.

I genuinely hope the file is correct now.

**Comment by ErikThiart** [comment]
@Timoon21 and @meijiao here is some additional support, on why it's incredibly important to protect our audiences, the bigger the support for this project the more important it becomes to be validated into the strict list. 

https://twitter.com/EricCryptoman/status/1729550680203862429

https://twitter.com/IcedKnife/status/1729974408838533384

**Comment by ErikThiart** [comment]
Here is the Solscan link: https://solscan.io/token/6DSqVXg9WLTWgz6LACqxN757QdHe1sCqkUfojWmxWtok for added information.

File: validated-tokens.csv [hunk 1]
```diff
@@ -824,3 +824,4 @@ SolTradingBot,STBOT,2x8o3hA5S5fBxCSE9hzVTf3RohcMWHqkDNKNEPuzprD5,9,https://yzevo
 Ponk,PONK,HeqCcMjmuV5s25J49YiJyT6bD5qWLkP88YPajBySniaV,5,https://bafkreieyllpkega3ytsmq5gdwx2p44roka5pnlclw2an4xivqmbq7prmge.ipfs.nftstorage.link/,true
 ClownOnSolana,$Clown,V7ntWk4D9Frwaec2CUjMRMMsJG28Cd31HxDKNdP1qjM,9,https://arweave.net/q1EkX68UfbUnvjpA87GaRjGxVaP6f0lYMBeQXmFzN6w,true
 Solabrador,SOBER,2ZE6hSL36e44wP168YMnxrbi1CSCFuD2BJm7NoNHfsmN,9,https://node2.irys.xyz/5xOYKlM26hYXX5wkZoQhc1yw0Bd0Xzoh_SU52TakOZQ,true
+Solanacorn,CORN,6DSqVXg9WLTWgz6LACqxN757QdHe1sCqkUfojWmxWtok,9,https://solanacorn.xyz/wp-content/uploads/2023/11/photo_2023-11-25_17-51-40.png,true
```
Comment (Timoon21) [inline validated-tokens.csv:827]:
```suggestion
Solanacorn,CORN,6DSqVXg9WLTWgz6LACqxN757QdHe1sCqkUfojWmxWtok,7,https://arweave.net/c_Tbuf4_HSFuogvTV1_FaqqbQipml9YlAETM941s178,true
```

**Review by Timoon21 — APPROVED** [review]

**Comment by ErikThiart** [comment]
@meijiao

**Review by benlvb — APPROVED** [review]

**Pull request merged**